core/common/src/main/java/alluxio/conf/Hash.java

```diff
@@ -50,6 +51,7 @@ public Hash(Supplier<Stream<byte[]>> properties) {
    */
   public void markOutdated() {
     mShouldUpdate.set(true);
+    mLastUpdateTime = System.currentTimeMillis();
```

Comment:
mLastUpdateTime should be put in the get() when it's actually updated
```
  public String get() {
    if (mShouldUpdate.get()) {
      synchronized (this) {
        // If another thread has recomputed the version, no need to recompute again.
        if (mShouldUpdate.get()) {
          mVersion = compute();
          mShouldUpdate.set(false);
        }
      }
    }
    return mVersion;
  }
}
```

Reply:
done